bin/functionMetadata_original.php

```diff
@@ -114,6 +115,9 @@
 	'random_bytes' => ['hasSideEffects' => true],
 	'random_int' => ['hasSideEffects' => true],
 
+	'ob_get_clean' => ['hasSideEffects' => true],
```

Comment:
@samsonasik you can use something like https://phpstan.org/r/9632df30-9ef5-4c50-b070-32972826e7d4 to test this change

the dumped type should be `false|string`

Reply:
`ob_start()` is required to make a `false|string` https://phpstan.org/r/1cff680a-416e-4ffb-adaf-db73a67ef759

Reply:
ob_start() could be anywhere, its not relevant for type analysis. Its global state. 

https://phpstan.org/r/a07431d0-a49d-47ff-bd7e-e432fec503f7